### ISSUE TITLE
Defined conditions not visible in edit mode for widget Catalog Products List

### ISSUE THREAD

**Issue by twiking** [magento/magento2#2913]
## Steps to repeat
1. Go to Admin -> Content -> Pages, and edit a page.
2. Add a new widget to the page
3. Select _Catalog Product Lists_ as the widget type.
4. Create a condition
5. Select Product attribute -> Category 
6. Click on ... and select a category
7. Insert the widget
8. Click on "Save and Continue Edit"
9. Double click on the widget (with WYSIWYG editor mode enabled)

**Expected result:** The defined conditions should be shown.
**Actual result:** The defined conditions are not shown.

The conditions are saved correctly. Confirmed by looking at the source

```
{{widget type="Magento\CatalogWidget\Block\Product\ProductsList" title="Product Top List" products_count="10" template="product/widget/content/grid.phtml" conditions_encoded="a:2:[i:1;a:4:[s:4:`type`;s:50:`Magento|CatalogWidget|Model|Rule|Condition|Combine`;s:10:`aggregator`;s:3:`all`;s:5:`value`;s:1:`1`;s:9:`new_child`;s:0:``;]s:4:`1--1`;a:4:[s:4:`type`;s:50:`Magento|CatalogWidget|Model|Rule|Condition|Product`;s:9:`attribute`;s:12:`category_ids`;s:8:`operator`;s:2:`==`;s:5:`value`;s:1:`6`;]]"}}
```

**User Agent:** Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.106 Safari/537.36.
**JS errors:** None

**Comment by shiftedreality** [comment]
@twiking, thank you for reporting!

Internal ticket MAGETWO-47858 was created

**Comment by duccanh** [comment]
lol

**Comment by fcapua-borngroup** [comment]
Is  there any update on this issue?

**Comment by facundocapua** [comment]
Is there any update on this issue?

**Comment by ghost** [comment]
Hi Is there any update on this?

**Comment by olysenko** [comment]
Hi, this issue is not reproducible(fixed) on develop branch. For now it is all update

**Comment by wigman** [comment]
Hello @olysenko,

I don't exactly get what you are saying. Is the problem solved in develop branch? I'm currently running Magento CE 2.1.2 in development mode and encounter the same problem.

**Comment by olysenko** [comment]
@wigman Yes this problem is solved in develop branch, but not in 2.1

**Comment by wigman** [comment]
@olysenko ok, great. would you happen to know the commit where it was fixed? That way I can cook up a patch. Thanks a lot!

**Comment by spyrule** [comment]
yes, please post a link to the known commit ... some of us cannot wait for the major patch release.

**Comment by gabs77** [comment]
Please, someone know the patch or the commit about that ?

**Comment by koenner01** [comment]
Can we please get a commit reference; Like @spyrule said, some of us CANNOT wait for the slow machine of the major releases.

This is a pressing issue and causing real problems for clients.

**Comment by koenner01** [comment]
Ok I looked into this further (because we cannot wait on official Magento response) and the commit where this issue is fixed is https://github.com/magento/magento2/commit/49d7b065efa

**Comment by kanduvisla** [comment]
Any update on this? It still persists in Magento 2.1.4

**Comment by leptoquark1** [comment]
Since this has a commit from 6 Jul 2016: When is this gonna be released?

**Comment by christian-forgacs** [comment]
Not released with Magento 2.1.6...

**Comment by dxhiep** [comment]
Hi @koenner01. I'm using your patch and it's work fine if is the first time. after save that and back to edit widget. The widget has shown conditions. but if you change conditions. The conditions not store. Please help me.

**Comment by kishorhase** [comment]
Conditions are actually saved, but does not restored when edit the widget.
 Any update on this? Issue still persist in Magento ver. 2.1.7

**Comment by bhargavlalo** [comment]
Yes same issue in Magento ver 2.1.7. Also condition not properly work on frontend. Any update on it?

**Comment by magento-engcom-team** [comment]
@twiking, thank you for your report.
We were not able to reproduce this issue by following the steps you provided. If you'd like to update it, please reopen the issue.
We tested the issue on 2.3.0-dev, 2.2.0, 2.1.9

**Comment by christian-forgacs** [comment]
@magento-engcom-team we can confirm, that the issue is fixed in our project with 2.1.9. thank you.
